Source/bmalloc/bmalloc/TZoneHeapManager.h

```diff
@@ -42,29 +42,22 @@
 
 namespace bmalloc { namespace api {
 
-static constexpr unsigned defaultBucketsForSmallSizes = 5;
-static constexpr unsigned defaultBucketsForLargeSizes = 3;
-static constexpr unsigned defaultMaxSmallSize = 128;
+#define TZONE_VERBOSE_DEBUG 0
+
+extern BEXPORT class TZoneHeapManager* tzoneHeapManager;
 
 class TZoneHeapManager {
-    enum State {
+    enum class State {
         Uninitialized,
         Seeded,
-        TypesRegistered
+        StartedRegisteringTypes
     };
```

Comment:
Nit: You could add `using enum State` and then you wouldn't need to prefix uses.

Reply:
I did this deliberately.  Will keep it this way because it's clearer that we're talking about the `State` everywhere we use this.